libraries/standard/mqtt/src/mqtt.c

```diff
@@ -293,6 +293,7 @@ static MQTTPubAckType_t getAckFromPacketType( uint8_t packetType )
 
         case MQTT_PACKET_TYPE_PUBCOMP:
         default:
+
             /* This function is only called after checking the type is one of
              * the above four values, so packet type must be PUBCOMP here. */
             assert( packetType == MQTT_PACKET_TYPE_PUBCOMP );
```

Comment:
Is this check redundant considering that there exists the case matching for `MQTT_PACKET_TYPE_PUBCOMP`  just before the default?